src/pages/project/project_wizard/EmptyProjectForm.tsx

```diff
@@ -35,10 +43,31 @@ export const EmptyProjectForm = ({ newProject, setNewProject, setShowProjectForm
           data-testid={dataTestId.newProjectPage.titleInput}
           variant="filled"
           onChange={(event) => setTitle(event.target.value)}
+          InputProps={{
+            endAdornment: titleSearch.isPending ? (
+              <CircularProgress size={20} />
+            ) : titleSearch.duplicateProject ? (
+              <ErrorIcon color="warning" />
+            ) : debouncedTitle && titleSearch.duplicateProject === undefined ? (
+              <CheckCircleIcon color="success" />
+            ) : undefined,
+          }}
           fullWidth
           placeholder={t('project.form.write_project_title')}
           label={t('common.title')}
         />
+        {titleSearch.duplicateProject && (
+          <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
+            <StyledInfoBanner>{t('project.duplicate_project_warning')}</StyledInfoBanner>
+            <ProjectListItem project={titleSearch.duplicateProject} />
+          </Box>
+        )}
+        {debouncedTitle && !titleSearch.isPending && titleSearch.duplicateProject === undefined && (
```

Comment:
Greit å sjekke på samme måte som på linje 59 tenker jeg, så vi vet at de to alltid er enige
```suggestion
        {debouncedTitle && !titleSearch.isPending && titleSearch.duplicateProject && (
```